script.js

```diff
@@ -81,7 +64,7 @@ function addGenerateCompareUrlButton() {
 		}
 		
 		firstSelectedCommitSHA = selectedCheckboxes[0].value;
-		secondSelectedCommitSHA = checkboxes[lastIndexOfSelectedCheckboxes + 1].value;
+		secondSelectedCommitSHA = selectedCheckboxes[1].value;
```

Comment:
Yes I recognize both of those issues but I want this behaviour anyway.

I´m thinking this is a settings in the future so that the user can change how they prefer.
I reverted back so that it behaves like before again.